swagger/body-dictionary.json

```diff
@@ -338,7 +368,7 @@
         "description": "Set dictionary value empty {\"0\": 1, \"1\": -1, \"2\": 3, \"3\": 300}",
         "parameters": [
           {
-            "name": "arrayBody",
+            "name": "dictBody",

```

Comment:
You might change the behavior of the tests and break people, please do not change the Swagger outside of adding the examples mark